src/liboslcomp/typecheck.cpp

```diff
@@ -2272,8 +2272,11 @@ OSLCompilerImpl::code_from_type (TypeSpec type) const
             out = 's';
         else if (elem == TypeDesc::NONE)
             out = 'x';
-        else
-            ASSERT (0);
+        else {
+            // error(ustring(), 0, "Possible compiler bug: %s(%s)",
```

Comment:
Remove this block of commented out code? (and maybe leave a comment at the end about how an empty string being returned signals an unknown type?)

Reply:
Updated.

Reply:
OK with this the latest update here?